### ISSUE TITLE
New version of opinel (0.24.1) breaks aws_recipes_assume_role.py

### ISSUE THREAD

**Issue by grettir** [nccgroup/AWS-recipes#4]
When I tried to run the `aws_recipes_assume_role.py` script this moring, I got the following error:

```
# ./aws_recipes_assume_role.py --profile xxxxxxx  --role-session xxxxxxx --role-arn arn:aws:iam::nnnnnnnnnnnnnn:role/xxxxxxx --debug

Traceback (most recent call last):
  File "./aws_recipes_assume_role.py", line 24, in main
    assume_role_and_save_in_credentials(args.profile[0], args.role_arn[0], args.role_session_name[0], args.mfa_serial[0], args.mfa_code[0], args.external_id[0])
TypeError: 'NoneType' object has no attribute '__getitem__'
```

The problem appears to be caused by the new version of the opinel package (0.24.1) that was released today.

If I downgrade to the previous version of opinel (0.24.0), it works again.

**Comment by l01cd3v** [comment]
Thanks ! I used a locally-modified repo when releasing, so the 0.24.1 package does not reflect the code on Github. This should be fixed with 0.24.2

**Comment by grettir** [comment]
Perfect, thanks!
